Automattic/newspack-plugin
# chore: remove Facebook Instant Articles as a recommended plugin

### All Submissions:






### Changes proposed in this Pull Request:



Removes the (no-longer-maintained) Facebook Instant Articles plugin from the Syndication wizard. #1393 removed FBIA from the WP Plugins page, but did not remove it from the Syndication wizard.

Also fixes the Distributor plugin info on the same page.

Also removes the Action Cards for Disqus, Disqus AMP, and Talk WP plugins from the Commenting wizard (these plugins were also removed in #1393).

Also adds error handling for the `PluginToggle` component.

### How to test the changes in this Pull Request:

1. View **Newspack > Syndication**. Confirm that there's no longer a card for Facebook Instant Articles.
2. Confirm that the card for Distributor is displayed correctly again (#1393 caused it to show a loading state forever).
3. To test the error handling, temporarily comment out or delete [this line](https://github.com/Automattic/newspack-plugin/pull/1466/files#diff-6a4458d03cc1b9a6a537660b28915359a74041ef4a907c61e8c36181a973eec3R296), then delete the Distributor plugin from your site. Attempt to toggle on Distributor and confirm you see an error message:

<img width="1047" alt="Screen Shot 2022-01-24 at 3 04 15 PM" src="https://user-images.githubusercontent.com/2230142/150872643-6ef74a6d-afe1-4d45-a048-de6777458eb7.png">

4. View **Newspack > Engagement > Commenting** and confirm there's just a single card for native WordPress Commenting here.

### Other information:
